core/services/registrysyncer/monitoring.go

```diff
@@ -16,12 +16,12 @@ var remoteRegistrySyncFailureCounter metric.Int64Counter
 var launcherFailureCounter metric.Int64Counter
 
 func initMonitoringResources() (err error) {
-	remoteRegistrySyncFailureCounter, err = beholder.GetMeter().Int64Counter("RemoteRegistrySyncFailure")
+	remoteRegistrySyncFailureCounter, err = beholder.GetMeter().Int64Counter("platform.registry_syncer.sync.failures")
```

Comment:
@patrickhuie19 IIUC all metrics should be prefixed by "platform"; is that for all metrics everywhere, or is "platform" just the prefix for Keystone related things?

Reply:
> just the prefix for Keystone related things
this 👍